content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting.rst

```diff
@@ -1,214 +1,99 @@
-==============================
-Subcontract your Manufacturing
-==============================
+=======================
+Subcontracting overview
+=======================
 
-Outsourcing a portion or all of your company’s manufacturing needs
-is not easy. To make it work correctly, you have to:
+.. |BoM| replace:: :abbr:`BoM (Bill of Materials)`
 
-- Manage the inventory of raw materials at your subcontractor
-- Ship raw material to your subcontractors, at the right time
-- Control incoming goods quality
-- Control subcontractors bills
+In manufacturing, subcontracting is the process of a company engaging a third-party manufacturer, or
+subcontractor, to manufacture products that are then sold by the contracting company.
 
-Here is an example of subcontracting the manufacturing of “C”, which is
-produced out of raw materials “A” and “B”.
+Subcontracting provides a variety of benefits for both the contracting company and the
+subcontractor.
 
-.. image:: subcontracting/subcontracting_01.png
-    :align: center
-    :alt:
+For the contracting company, subcontracting allows them to sell a wide variety of manufactured
+products without having to worry about investing in and maintaining the equipment and labor required
+to handle the manufacturing themselves.
 
-With its MRP subcontracting feature, Odoo helps you handle this flow easily.
+This helps contracting companies stay flexible throughout economic cycles, as they can easily
+increase or decrease their engagements with subcontractors as necessitated by the current moment. It
+also means that they are able to focus on the tasks they excel at, while delegating more specialized
+work to subcontractors better suited to handling it.
```

Comment:
This is useful background information for understanding subcontracting as a concept, but I'm not sure if it's necessary in a guide for understanding how subcontracting works in Odoo. If Odoo has a feature that assists with flexibility through economic cycles, it would be helpful to link it here.

Otherwise, I'd actually move up your explanation about the different kinds of subcontracting workflows in Odoo up here, so readers can select and read relevant docs.